packages/components/managed-data-table/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './src';
```

Comment:
```suggestion
// This file exists  because we want jest to use our non-compiled code to run tests
// if this file is missing, and you have a `module` or `main` that points to a non-existing file
// (ie, a bundle that hasn't been built yet) then jest will fail if the bundle is not yet built.
// all apps should export all their named exports from their root index.js
export { default } from './src';
export * from './src';
```